scripts/r.bat

```diff
@@ -0,0 +1,9 @@
+@echo off
```

Comment:
at the moment if you just run `r.bat` you get
`error: `ruff <path>` has been removed. Use `ruff check <path>` instead.` 

i think we should check the arg count and print out `ruff --help` or print out the options being format or check